pulsar-broker/src/test/java/org/apache/pulsar/client/api/SimpleProducerConsumerTest.java

```diff
@@ -3557,7 +3557,8 @@ public void testMultiTopicsConsumerImplPauseForManualSubscription() throws Excep
         }
 
         // 6. should not consume any messages
-        Awaitility.await().untilAsserted(() -> assertNull(consumer.receive(RECEIVE_TIMEOUT_SECONDS, TimeUnit.SECONDS)));
+        Message<byte[]> msg = consumer.receive(2, TimeUnit.SECONDS);
```

Comment:
there's already several constants for timeouts. Perhaps `receive(RECEIVE_TIMEOUT_MEDIUM_MILLIS, TimeUnit.MILLISECONDS)` could be used instead.